.binder/environment.yml

```diff
@@ -54,9 +54,10 @@ dependencies:
   - python >=3.10,<3.11
   # build
   - doit >=0.34,<1
-  - flit >=3.7.1,<4
+  - hatch >=1.6.3,<2
   - jsonschema >=3
   - jupyter-server-mathjax >=0.2.3
+  - jupyterlab >=3.5.3,<3.6
```

Comment:
perhaps remove the pin above in the `demo` section so it isn't duplicated (without refining)